src/client/client.js

```diff
@@ -18,6 +18,9 @@
  * Main App
  * Set up the global store and kick off rendering
  */
+
+// polyfill for ie 11
+import 'abort-controller/polyfill';
```

Comment:
does this automatically only get included in the legacy bundle?

Reply:
@anescobar1991 yeah it only works on branches from the repo itself, not on forks.